mautic/mautic
# Maut 3476 optionlist vs list

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Optionlist does not work

[//]: # ( As applicable: )
#### Steps to reproduce the bug:

1. Create a select-multiple custom field with options
2. Create a form field that maps to the new custom field, set Use Assigned Contact Field Values to No & add fields
3. Switch 'Use assigned...' to Yes.
4. Save and close
5. And then change 'Use assigned...' to No.
6. You still see assigned values

#### Steps to test this PR:
0. Load up [this PR](https://mautibox.com)
1. How to reproduce the bug:
2. Create a select-multiple custom field with options
3. Create a form field that maps to the new custom field, set Use Assigned Contact Field Values to No & add fields
4. Switch 'Use assigned...' to Yes.
5. Save and close
6. And then change 'Use assigned...' to No.
7. You should see your custom values

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
